src/app/sandbox/mapbox/page.tsx

```diff
@@ -0,0 +1,36 @@
+"use client";
+import Map, { Layer, Source } from "react-map-gl";
+import "mapbox-gl/dist/mapbox-gl.css";
+
+function MapboxTest() {
+  return (
+    <Map
+      mapboxAccessToken="pk.eyJ1IjoiZGV2c2VlZCIsImEiOiJjbHo5NWh0d3QwMDU0MnBvanAwMmRzYWl0In0.lLN2Ur91TbYhV520kQ3SrA"
+      initialViewState={{
+        longitude: 0,
+        latitude: 0,
+        zoom: 2,
+      }}
+      style={{ width: 1000, height: 800 }}
+      mapStyle="mapbox://styles/mapbox/light-v11"
```

Comment:
@oliverroick can we try not using the basemap from Mapbox and just the MVT boundaries from the local endpoint?